tests/unit/views/Dashboard.spec.js

```diff
@@ -36,38 +36,38 @@ describe('Dashboard.vue', () => {
 
   describe('for saphyr', () => {
     beforeEach(() => {
-      box = wrapper.find('.card.saphyr')
+      box = wrapper.find('[data-pipeline=Saphyr]')
     })
 
     it('will have a title', () => {
-      expect(box.find('.card-title').text()).toEqual('Saphyr')
+      expect(box.find('[data-attribute=title]').text()).toEqual('Saphyr')
     })
 
     it('will have a description', () => {
-      expect(box.find('.card-text').text()).toBeDefined()
+      expect(box.find('[data-attribute=description]').text()).toBeDefined()
     })
 
     describe('route buttons', () => {
       it('will have a reception button', () => {
-        let receptionButton = box.findAll('button').at(0)
+        let receptionButton = box.findAll('a').at(0)
```

Comment:
How come findAll `a` ?

Reply:
We changed the buttons to hyperlinks and it broke the tests. It looks weird but I think this is the standard way in vue test utils to find stuff. I couldn't find a better way.